sdk-core/src/main/java/dev/restate/sdk/core/InvocationStateMachine.java

```diff
@@ -735,7 +743,7 @@ public String toString() {
         + serviceName
         + '\''
         + ", state="
```

Comment:
Nit: maybe rename the string rendering to match the field name:
```suggestion
        + ", invocationState="
```